autoload/neomake/makers/ft/rust.vim

```diff
@@ -21,6 +21,33 @@ function! neomake#makers#ft#rust#rustc() abort
         \ }
 endfunction
 
+function! neomake#makers#ft#rust#cargotest() abort
+    let maker = {
+        \ 'exe': 'cargo',
+        \ 'args': ['test', '--quiet'],
+        \ 'auto_enabled': 1,
```

Comment:
`auto_enabled` does not belong here.
Why have you added it?
It gets internally to see if a maker was configured/called explicitly, or via `GetEnabledMakers`.

Reply:
I think I took it from what NeomakeCheck was displaying (something like what's displayed at #1886. But ok, I'll remove that :)